inc/shortcodes/class-pdf.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Shortcake_Bakery\Shortcodes;
+
+class PDF extends Shortcode {
+
+	public static function get_shortcode_ui_args() {
+		return array(
+			'label'          => 'PDF',
+			'listItemImage'  => 'dashicons-media-document',
+			'attrs'          => array(
+				array(
+					'label'  => 'URL',
+					'attr'   => 'url',
+					'type'   => 'text',
+				),
+			),
+		);
+	}
+
+	public static function callback( $attrs, $content = '' ) {
+		if ( empty( $attrs['url'] ) ) {
+			return '';
+		}
+
+		$url = esc_url_raw( $attrs['url'] );
+		$ext = pathinfo( $url, PATHINFO_EXTENSION );
+		if ( 'pdf' !== strtolower( $ext ) ) {
+			return '';
+		}
+		return '<iframe class="bakery-responsive" data-true-height="800px" data-true-width="600px" width="600px" height="800px" frameBorder="0" src="' . esc_url( 'https://mozilla.github.io/pdf.js/web/viewer.html?file=' . rawurlencode( $url ) ) . '"></iframe>';
```

Comment:
Can we use `shortcake-bakery-responsive`, and `shortcake-bakery` generally as our slug everywhere?